ENIGMAsystem/SHELL/Graphics_Systems/OpenGL-Common/shared.cpp

```diff
@@ -7,3 +7,4 @@
 #include "shader.cpp"
 #include "surface.cpp"
 #include "vertex.cpp"
+#include "types.cpp"
```

Comment:
`texture_copy.cpp`?

Reply:
no cause it not supported in all. i manually include it where it is supported